src/utils/textStreamer.tsx

```
@@ -0,0 +1,22 @@
+        if (currentIndex < fullText.length) {
+          const chunk = fullText.slice(currentIndex, currentIndex + chunkSize);
+          onUpdate(chunk); // Notify the caller with the new chunk
+          currentIndex += chunkSize;
+  
+          setTimeout(processChunk, delayMs); // Recursive call with delay
+        }
+      };
+  
+      processChunk();
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)